### DISCUSSION TITLE
Snap version of dbeaver unable to talk to ssh-agent

### DISCUSSION THREAD

**Discussion by ferrybig** [dbeaver/dbeaver#18767] [DBeaver Development]
### Description

I used the snap version of dbeaver becaus of it sautomatic updates. I expectd the same features in the snap version as in the release version.

It will not see that ssh-agent is running, as ssh-agent places its files in /tmp, while snap apps have a separate /tmp folder. 

### DBeaver Version

Version 22.2.1.202209251341

### Operating System

Ubuntu 22.04.1 LTS

### Database and driver

SSHJ

### Steps to reproduce

1. start ssh-agent in a terminal via `eval \`ssh-agent\``
2. Add your keys to ssh-agnt via `ssh-add`
3. start dbeaver n the same terminal via /snap/bin/dbeaver-ce
4. Make a new connection, choose any DB, like MySQL, postgress of SQlite
5. Go to the ssh properties
6. Select the SSH tab
7. Fill in a valid SSH server
8. Select authentication agent
9. Press test connection

Expected:
It connects to the server with your ssh-agent credentials

Actual:
It gives an error that it cannot talk to the ssh-agent

### Additional context

Manually using public key is not a good workaround, as every restart of the problem, dbeaver will say your private key cannot be found and you will have to add it again, another side effect of the "good" isolation of snap...

**Comment by Destrolaric** [comment]
Thank for the bug report!

**Comment by riednyko** [comment]
https://github.com/dbeaver/dbeaver/issues/8800
